frontend/src/components/Navbar/Sidebar/MobileSidebar/mobileSidebar.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { LastWillContextProvider } from '../../../../store/last-will/LastWillContext'
+import { MobileSidebar } from './MobileSidebar'
+import { SidebarProps } from '../Sidebar'
+
+const meta: Meta<typeof MobileSidebar> = {
+	title: 'Design System/Navbar/MobileSideBar',
+	component: MobileSidebar,
```

Comment:
```suggestion
	component: MobileSidebar,
	parameters: {
		viewport: {
			defaultViewport: 'mobile2',
		},
	},
```
sonst hidet es alles in der story weil der screen ja auf desktop by default gesetzt ist^^

Reply:
very nice